draft-ietf-tls-tls13.md

```diff
@@ -3302,8 +3303,9 @@ message.
 in the sender's end-entity certificate. RSA signatures MUST use an
 RSASSA-PSS algorithm, regardless of whether RSASSA-PKCS-v1_5 algorithms
 appear in "signature_algorithms". SHA-1 MUST NOT be used in any signatures in
-CertificateVerify. (Note that rsa_pkcs1_sha1 and dsa_sha1, the only defined
-SHA-1 signature algorithms, are undefined for CertificateVerify signatures.)
+CertificateVerify. (Note that all SHA-1 signature algorithms in this
+specification are defined solely for use in certificates, not CertificateVerify
+signatures.)
```

Comment:
I suggest also simplifying/clarifying this slightly while we're here. This doesn't really need to be in parenthesis, either. (minor preexisting nit)

Suggested replacement:

All SHA-1 signature algorithms in this specification are defined solely for use in legacy certificates, and are not valid for CertificateVerify signatures.